CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🐞 A use after free bug would sometimes crash the node while it was shutting
+  down. [#896](https://github.com/tenzir/vast/pull/896)
```

Comment:
nit: The bug didn't crash the node, it was our asan instrumentation that did ;)

Reply:
Technically true, but that is just luck, the memory location could have been given back to the OS. Also I don't want to get too technical in the changelog.